app/controllers/users/omniauth_callbacks_controller.rb

```diff
@@ -0,0 +1,20 @@
+require "open-uri"
+require "json"
+
+class Users::OmniauthCallbacksController < Devise::OmniauthCallbacksController
+  def github
+    user = User.find_for_github_oauth(request.env['omniauth.auth'])
+    if user == false
+      session['devise.github_data'] = request.env['omniauth.auth']
+      set_flash_message(:alert, :not_a_wagon_student, kind: 'Github')
```

Comment:
What is this `:not_a_wagon_student` about? Do we need that?

Reply:
No. I removed it.